images/router/haproxy/reload-haproxy

```diff
@@ -73,8 +73,13 @@ retries=20
 
 
 # sort the path based map files for the haproxy map_beg function
+# Leaving any wildcard route at the end of the resulting file,
+# See https://github.com/openshift/origin/issues/16724 for more details.
 for mapfile in "$haproxy_conf_dir"/*.map; do
-  sort -r "$mapfile" -o "$mapfile"
+  grep -v -F '^[^\.]*\' "$mapfile" | sort -r -o /tmp/"$mapfile"
+  grep -F '^[^\.]*\' "$mapfile" >> /tmp/"$mapfile"
+  cp /tmp/"$mapfile" "$haproxy_conf_dir"/"$mapfile"
+  rm /tmp/"$mapfile"
```

Comment:
Could use:  `mv -f "/tmp/$mapfile" "$haproxy_conf_dir"/"$mapfile"` to collapse the 2 commands.